src/components/tickets/ticket.routes.ts

```diff
@@ -33,7 +33,7 @@ router.put('/:id',
   ],
   handleValidationError(400),
   moveTicket, (_req, res) =>
-    res.sendStatus(201)
+    res.json(STATUSES.get(_req.body.status))
```

Comment:
We generally use leading underscores to indicate a parameter that we don't use. To keep this consistent, we should change `_req` to `req` and use it like that.